WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalProgressDisplaying.swift

```diff
@@ -0,0 +1,39 @@
+import UIKit
+
+protocol CardPresentModalProgressDisplaying: CardPresentPaymentsModalViewModel {
```

Comment:
This part doesn't seem related to errors or mentioned in the description/testing, I'm wondering if this was intentionally included

Reply:
Github's being a bit strange.

Here's the first four commits from this PR:
<img width="1307" alt="CleanShot 2022-12-07 at 13 03 04@2x" src="https://user-images.githubusercontent.com/2472348/206186395-b856a977-ca48-4bbe-9557-4535eb7b69e8.png">

And here's the three commits from the previous PR, which is merged to trunk:
<img width="1302" alt="CleanShot 2022-12-07 at 13 03 54@2x" src="https://user-images.githubusercontent.com/2472348/206186493-737ed55e-7882-48be-9c55-7bdbf934063a.png">

So... they're all already in trunk with matching hashes, so I'm pretty sure this will sort itself out when I merge this PR... but I will check carefully! Thanks for the heads-up.